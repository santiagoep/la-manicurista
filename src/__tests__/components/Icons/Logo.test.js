import React from "react";

import render from "../../render";
import Logo from "@components/Icons/Logo";

describe("<Logo />", () => {
  it("should render snapshot", () => {
    const { renderer } = render({ theme: true });
    const component = renderer(<Logo />);
    expect(component).toMatchSnapshot();
  });
});
