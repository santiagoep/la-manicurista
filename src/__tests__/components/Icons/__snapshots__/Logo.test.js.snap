// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render snapshot 1`] = `
<img
  alt="LaManicurista Logo"
  className="Logo__StyledImage-sc-17vxmqw-0 gCRnca"
  src="https://lamanicurista.com/wp-content/uploads/2019/01/Logo-La-manicurista-1.png"
/>
`;
